Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -3185,10 +3221,35 @@
       Dead:
         Base: cat2_dead
 
+- type: entity
+  id: MobSyndicateBaseExplosive
+  abstract: true
+  components:
+  - type: GhostTakeoverAvailable
+  - type: AutoImplant
+    implants:
+    - MicroBombImplant
+  - type: ExplosionResistance
+    damageCoefficient: 0.2
+  - type: NpcFactionMember
+    factions:
+    - Syndicate
+  - type: MeleeWeapon
+    altDisarm: false
+    angle: 0
+    animation: WeaponArcBite
+    soundHit:
+      path: /Audio/Effects/bite.ogg
+    damage:
```

Comment:
bombroach parents off mothroach first so only syndicat uses it
this can just be put back on the syndicat